cmake/projects/libuv/hunter.cmake

```diff
@@ -0,0 +1,26 @@
+# libuv
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+# Load used modules
+include(hunter_add_version)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+# List of versions here...
+hunter_add_version(
+  PACKAGE_NAME
+  libuv
+  VERSION
+  1.8.0
+  URL
+  https://github.com/akalsi87/libuv-cmake/archive/libuv-cmake-1.8.0.tar.gz
```

Comment:
I've created hunter-packages repository:
* https://github.com/hunter-packages/libuv

Please use it. I've send invitation to team (check your email).